tests/functional/issues-auth.js

```diff
@@ -27,34 +27,6 @@ define(
         return FunctionalHelpers.logout(this);
       },
 
-      "Closing and reopening an issue": function() {
```

Comment:
note to self: We should add a test like this in #1747 -- testing that changing a milestone from an open to a closed one updates the UI/state of the issue.